recipes/onetbb/all/conanfile.py

```diff
@@ -0,0 +1,152 @@
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/oneapi-src/oneTBB"
+    description = (
+        "oneAPI Threading Building Blocks (oneTBB) lets you easily write parallel C++"
+        " programs that take full advantage of multicore performance, that are portable, composable"
+        " and have future-proof scalability.")
+    topics = ("tbb", "threading", "parallelism", "tbbmalloc")
+    provides = "tbb",
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "tbbmalloc": [True, False],
+        "tbbproxy": [True, False],
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+        "tbbmalloc": False,
+        "tbbproxy": False,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if tools.Version(self.version) < "2021.2.0":
+            del self.options.shared
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.get_safe("shared", True):
+            del self.options.fPIC
+
+    def validate(self):
+        if (self.settings.os == "Macos"
+                and self.settings.compiler == "apple-clang"
+                and tools.Version(self.settings.compiler.version) < "8.0"):
+            raise ConanInvalidConfiguration(
+                "{} {} couldn't be built by apple-clang < 8.0".format(
+                    self.name,
+                    self.version,
+                ))
+        if not self.options.get_safe("shared", True):
+            self.output.warn(
+                "oneTBB strongly discourages usage of static linkage")
+        if (self.options.tbbproxy
+                and not (self.options.tbbmalloc
+                         and self.options.get_safe("shared", True))):
+            raise ConanInvalidConfiguration(
+                "tbbproxy needs tbbmalloc and shared options")
+
+    def package_id(self):
+        del self.info.options.tbbmalloc
+        del self.info.options.tbbproxy
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            strip_root=True,
+            destination="src",
+        )
+
+    def generate(self):
+        toolchain = CMakeToolchain(self)
+        toolchain.variables["TBB_TEST"] = False
+        toolchain.variables["CMAKE_INSTALL_BINDIR"] = "bin"
+        toolchain.variables["CMAKE_INSTALL_SBINDIR"] = "bin"
+        toolchain.variables["CMAKE_INSTALL_LIBEXECDIR"] = "bin"
+        toolchain.variables["CMAKE_INSTALL_LIBDIR"] = "lib"
+        toolchain.variables["CMAKE_INSTALL_INCLUDEDIR"] = "include"
+        toolchain.variables["CMAKE_INSTALL_OLDINCLUDEDIR"] = "include"
+        toolchain.variables["CMAKE_INSTALL_DATAROOTDIR"] = "share"
+        toolchain.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder="src")
```

Comment:
Should be `build_script_folder ` due to the recent commit https://github.com/conan-io/conan/commit/db9db58f9e481e697ae451d9d72399c81135043a

```suggestion
        cmake.configure(build_script_folder ="src")
```